packages/components/src/Tabs/Tabs.tsx

```diff
@@ -82,16 +82,14 @@ export function Tabs({ children, defaultTab = 0, onTabChange }: TabsProps) {
     <div className={styles.tabs}>
       <div className={overflowClassNames}>
         <ul role="tablist" className={styles.tabRow} ref={tabRow}>
-          <li role="presentation">
-            {React.Children.map(children, (tab, index) => (
-              <InternalTab
-                label={tab.props.label}
-                selected={activeTab === index}
-                activateTab={activateTab(index)}
-                onClick={tab.props.onClick}
-              />
-            ))}
-          </li>
+          {React.Children.map(children, (tab, index) => (
```

Comment:
Moving the `li` into `InternalTab` allows `gap` to space the tabs out evenly like there are in SG-1